src/jira/client/jira-client-issue-key-helper.ts

```diff
@@ -0,0 +1,172 @@
+import { uniq } from "lodash";
```

Comment:
ooooo again a lot of these are just pulled from JC and dumped in here, I've done a bunch of house keeping, more typings but still not quite done.

will highlight any major changes